cookie-banner-rules-list.json

```diff
@@ -636,6 +636,265 @@
       },
       "id": "c5243e7c-eb86-4a9d-947c-7129e99fbd72",
       "last_modified": 1660836869997
-    }
+    },
+    	{
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-consent-sdk"
+  },
+  "domain": "cloudflare.com",
+  "schema": 1665622072973,
```

Comment:
This looks more like an internal remote settings field, which we don't need in this repo. However we already have existing entries with it. Let's discuss this in the next meeting and see if we want to remove the field from the list.

Reply:
I don't think we've touched on this subject in the meeting, do we wan to add the new rules with the `schema` field?

Reply:
I don't think we need the `schema` field here, remote settings will add it internally. Let's leave it out.